aead/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+//! A set of traits designed to support authenticated encryption.
+
+#![no_std]
+
+extern crate alloc;
+
+use alloc::vec::Vec;
+use generic_array::{GenericArray, ArrayLength};
```

Comment:
It'd probably be good to have a `pub use generic_array` like the other trait crates